app/src/main/res/values/strings.xml

```diff
@@ -8,4 +8,7 @@
     <string name="curr_speed_label">Current</string>
     <string name="avg_speed_label">Average</string>
     <string name="initial_speed">0.0 m/s</string>
+    <string name="TitleLocation">Activate location</string>
+    <string name="TextLocation">You have to activate the location to perform a test</string>
```

Comment:
Shouldn't it say 'enable location to perform the test' ?
I'm never sure with the articles though, we don't have those in Polish :sweat_smile:

Reply:
Depends if you speak about test in general or this precise test. I prefer a in this case, because it's for all test in general